packages/webpack-loaders/src/gen-css-bundles.js

```diff
@@ -0,0 +1,95 @@
+import { assign } from "lodash";
```

Comment:
I know you're planning to clean all this up - but wanted to point out the filename in case you miss it :)